StardewArchipelago/Items/ItemParser.cs

```diff
@@ -50,7 +50,7 @@ public bool TrySendItemImmediately(ReceivedItem receivedItem)
         public LetterAttachment ProcessItemAsLetter(ReceivedItem receivedItem)
         {
             var itemIsResourcePack = TryParseResourcePack(receivedItem.ItemName, out var stardewItemName, out var resourcePackAmount);
-            var itemName = receivedItem.ItemName;
+            var itemName = _nameMapper.GetInternalName(receivedItem.ItemName);
```

Comment:
the block about resource packs does not use the mapped internal name... is that okay?